python/BUILDING.md

```diff
@@ -16,13 +16,49 @@ important differences:
 
     `$ make python_wheel`
 
-After this procedure completes, the `ngraph/build/python/dist` directory should 
-contain the Python packages of the version you cloned. For example, if you 
-checked out and built `0.21` for Python 3.7, you might see something like: 
+After this procedure completes, the `ngraph/build/python/dist` directory should
+contain the Python packages of the version you cloned. For example, if you
+checked out and built `0.21` for Python 3.7, you might see something like:
 
     $ ls python/dist/
-    ngraph-core-0.21.0rc0.tar.gz  
-    ngraph_core-0.21.0rc0-cp37-cp37m-linux_x86_64.whl  
+    ngraph-core-0.21.0rc0.tar.gz
+    ngraph_core-0.21.0rc0-cp37-cp37m-linux_x86_64.whl
+
+## Building nGraph Python Wheels on Windows
+
+It's now possible to build the python wheel on Windows too. The build process consists of 3 steps:
```

Comment:
> It's now possible to build the python wheel on Windows too.

This sentence is not future proof. Let's not make the docs sound as news.
I would recommend removing the sentence.